crates/rpc/rpc/src/eth/error.rs

```diff
@@ -154,7 +154,19 @@ pub enum InvalidTransactionError {
     SenderNoEOA,
     /// Thrown during estimate if caller has insufficient funds to cover the tx.
     #[error("Out of gas: gas required exceeds allowance: {0:?}")]
-    OutOfGas(U256),
+    BasicOutOfGas(U256),
+    /// TODO?
+    //#[error("Out of gas:")]
+    //MemoryLimitOutOfGas(U256),
```

Comment:
It is really necessary to handle this error? Do the reth's revm have memory expansion limits?